views/index.ejs

```diff
@@ -49,6 +49,7 @@
           <input type="submit" value="" class="search_form__action" onclick="submitSearch()" title="<%= _('Search') %>">
         </div>
         <div id="react_menu__container"></div>
+        <button class="search_form__direction_shortcut" title="<%= _('Directions', 'top bar') %>"></button>
```

Comment:
What if Directions are disabled ?  
Should this button be disabled ? or hidden ?

Reply:
I think the best solution is to hide it. I'll do it, thanks!

Reply:
Done!

|Enabled|Disabled|
|---|---|
|![Capture d’écran de 2020-07-09 14-46-57](https://user-images.githubusercontent.com/243653/87041833-19114f80-c1f3-11ea-8d2d-a6a3f0a652c9.png)|![Capture d’écran de 2020-07-09 14-46-01](https://user-images.githubusercontent.com/243653/87041831-1878b900-c1f3-11ea-929c-ece3b8aa72d3.png)|